frankenphp_test.go

```diff
@@ -100,6 +100,20 @@ func testHelloWorld(t *testing.T, opts *testOptions) {
 	}, opts)
 }
 
+func TestFinishRequest_module(t *testing.T) { testFinishRequest(t, nil) }
+//func TestFinishRequest_worker(t *testing.T) { testFinishRequest(t, &testOptions{workerScript: "finish-request.php"}) }
```

Comment:
This goes into an infinite loop, and I'm not sure why yet. It appears to work fine "in real life" but there must be something wrong. I'll dig into it.